src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -295,6 +296,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             {
                 // Preview features.
                 case MessageID.IDS_FeatureLambdaDiscardParameters: // semantic check
+                case MessageID.IDS_FeatureMemberNotNull:
```

Comment:
>IDS_FeatureMemberNotNull [](start = 31, length = 24)

Not sure why this is conditioned on a language version.  Improvements and changes in warnings for nullable was explicitly one of the things we gave ourselves permission to change over time without a language version change.